app/controllers/claims/support/schools/claims/mentors_controller.rb

```diff
@@ -1,20 +1,24 @@
-class Claims::Support::Schools::Claims::MentorsController < Claims::ApplicationController
+class Claims::Support::Schools::Claims::MentorsController < Claims::Support::ApplicationController
```

Comment:
Found a bug from this work. Support controller inheriting from the non-support application controller.